npm/test/updatefilecheckprogress.test.ts

```diff
@@ -0,0 +1,148 @@
+const mockFileCheckProcessing = {
```

Comment:
It looks like you're copying what's already there, but should we be formatting the test names in the same way as the main TypeScript file names? The file names use kebab-case, whereas in the tests we're puttingeverythingintooneword.

Reply:
Yes, I was just following convention but I can add give it the same/similar name. should I do it just for the file checks files and then create another PR where I rename the others?

Reply:
If you could rename any that you're working with directly, that would be great. We can update the other ones as we go if it's too much effort to rename them all!